### PR TITLE
adapt logging, add missing sample types

### PR DESCRIPTION
**Pull request by harrypuuter**
needed for https://github.com/KIT-CMS/CROWN/pull/193

### CODE REVIEW

File: producers/event.py [hunk 1]
```diff
@@ -93,6 +93,62 @@
...
+    output=[q.is_embedding_mc],
+    scopes=["global"],
+)
+is_embedding_mc = Producer(
+    name="is_embedding_mc",
+    call="basefunctions::DefineQuantity({df}, {output}, {is_embedding_mc})",
+    input=[],
+    output=[q.is_embedding_mc],
+    scopes=["global"],
+)
```
Comment (nshadskiy) [inline producers/event.py:130]: `is_embedding_mc` is defined twice.

File: quantities/output.py [hunk 2]
```diff
@@ -232,6 +232,15 @@
 is_ggh_htautau = Quantity("is_ggh_htautau")
 is_vbf_htautau = Quantity("is_vbf_htautau")
 is_diboson = Quantity("is_diboson")
+is_vbf_hbb = Quantity("is_vbf_hbb")
+is_ggh_hbb = Quantity("is_ggh_hbb")
+is_rem_hbb = Quantity("is_rem_hbb")
+is_embedding_mc = Quantity("is_embedding_mc")
+is_ggh_hbb = Quantity("is_ggh_hbb")
```
Comment (nshadskiy) [inline quantities/output.py:239]: `is_ggh_hbb` is defined twice.

File: producers/event.py (hunk 1 above)
Comment (harrypuuter) [reply producers/event.py]: solved

File: quantities/output.py (hunk 2 above)
Comment (harrypuuter) [reply quantities/output.py]: solved

**Review by nshadskiy — APPROVED** [review]
Looks good to me!

**Pull request merged**